pkg/collector/corechecks/cluster/kubernetesapiserver/events_common.go

```diff
@@ -267,6 +267,33 @@ func getInvolvedObjectTags(involvedObject v1.ObjectReference, taggerInstance tag
 		}
 	}
 
+	var entityID string
+
+	switch involvedObject.Kind {
+	case podKind:
+		entityID = fmt.Sprintf("kubernetes_pod_uid://%s", involvedObject.UID)
```

Comment:
Not urgent at all, but just as a headsup:

Thanks to [this](https://github.com/DataDog/datadog-agent/pull/28645) PR, we will be able to create the entity ids with the NewEntityID function exposed by the tagger, which can avoid typos in the id's and also have a performance impact.

You will be able to create the entity id as follows:
`types.NewEntityID(types.KubernetesPodUID, involvedObjects.UID)` 

[Here](https://github.com/DataDog/datadog-agent/blob/ba27a220b174de70d63d38302067f1c9b70f3358/pkg/process/util/containers/containers.go#L122) is an example.

This applies to all the lines below where you create the id as a string with `fmt.Sprintf`

I'd suggest one of 2 options:
- Not waiting for the other PR to be merged, and merging your PR as it is, but create another card (in CONINT board or CONTP board) to initialise the entity id here with the NewEntityID function. We can do it later when the other PR is merged.
- Don't merge your PR for now, and wait some time until the other PR is merged, then modify your code and merge it. The other PR should be merged very soon, I think tomorrow in the worst case. 

If you have any questions, feel free to ping me.